{{ cookiecutter.project_slug }}/api/pyproject.toml

```diff
@@ -0,0 +1,102 @@
+[tool.pytest.ini_options]
+DJANGO_SETTINGS_MODULE = "{{ cookiecutter.project_slug }}.settings"
+python_files = ["tests.py", "test_*.py", "*_test.py"]
+
+[tool.black]
+line-length = 120
+target-version = ['py310']
+exclude = '''
+/(
+    \.eggs
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.env
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+  | node_modules
+)/
+'''
+
+[tool.bandit]
+skips = ['B101',]
+
+[tool.isort]
+profile = 'black'
+line_length=120
+multi_line_output = 3
+lines_between_types = 1
+lines_after_imports = 2
+known_first_party='{{ cookiecutter.project_slug }}'
+
```

Comment:
Django proposes following imports style https://docs.djangoproject.com/en/4.2/internals/contributing/writing-code/coding-style/#imports. I we want to follow this style need to add the following options:

known_django = ['django']
sections= ['FUTURE', 'STDLIB', 'THIRDPARTY', 'DJANGO', 'FIRSTPARTY', 'LOCALFOLDER']

Reply:
This makes more sense for me.